pkg/config/config.go

```diff
@@ -82,6 +84,10 @@ func (c *Config) EnsureConfigValid() {
 	}
 }
 
+func (c *Config) UpdateRedisPassword(val string) {
+	c.RedisPassword = val
+}
+
 func GetConf() Config {
 	if GlobalConfig == nil {
 		return getDefaultConfig()
```

Comment:
There is a slight difference between the two versions of the code you provided. The first version contains an empty string literal `SecretEncryptKey` which could lead to some unexpected behavior if used elsewhere. In this case, it's not specified what it means exactly in the context of `config`, so I would recommend using a more precise name instead (e.g., SecretEncryptionKey). 

Additionally, there was no previous implementation for RedisPassword being set in UpdateRedisPassword function.

This small refinement can help avoid any possible bugs or confusion related to these changes.